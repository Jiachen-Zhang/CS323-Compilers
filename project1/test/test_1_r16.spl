int test_1_r16()
{
  int dec_int = 323; // inline comment
}
