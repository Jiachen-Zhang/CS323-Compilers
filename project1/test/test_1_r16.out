Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: test_1_r16
        LP
        RP
      CompSt (2)
        LC
        DefList (3)
          Def (3)
            Specifier (3)
              TYPE: int
            DecList (3)
              Dec (3)
                VarDec (3)
                  ID: dec_int
                ASSIGN
                Exp (3)
                  INT: 323
            SEMI
        RC
